group_vars/all

```diff
@@ -15,3 +15,21 @@ oms_prod_repo: /var/www/html/omsprod/
 oms_dev_repo: /var/www/html/omsdev/
 # OMS
 oms_git: https://github.com/MykolaKr/OMS.git
+#MySQL
+#replication: yes
+mysql_replication_master: db1
+replic_user: replic_user
+replic_userpassword: 123
+replic_db: omsdb
+# A list of the mysql DB to be created
+oms_mysql_db:                              
+  - name: omsdb
+oms_mysql_users:                              
+  - name: 056db
+    host: "%"
+    pass: 056db
+    priv: "omsdb.*:ALL"
+  - name: 055db
+    host: "%"
+    pass: 055db
+    priv: "omsdb.*:ALL"
```

Comment:
поки є дві ролі секвела і ми не об"єднили.... в групу олл!

2016-04-17 11:26 GMT+03:00 Iaroslav Pleten <notifications@github.com>:

> In group_vars/all
> <https://github.com/IF-061-DevOps/OMS-ansible/pull/89#discussion_r59980214>
> :
>
> > +mysql_replication_master: db1
> > +replic_user: replic_user
> > +replic_userpassword: 123
> > +replic_db: omsdb
> > +# A list of the mysql DB to be created
> > +oms_mysql_db:
> > +  - name: omsdb
> > +oms_mysql_users:
> > +  - name: 056db
> > +    host: "%"
> > +    pass: 056db
> > +    priv: "omsdb.*:ALL"
> > +  - name: 055db
> > +    host: "%"
> > +    pass: 055db
> > +    priv: "omsdb.*:ALL"
>
> Група All? Может db?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IF-061-DevOps/OMS-ansible/pull/89/files/355404b0d42e34a44767b8cc94d4a96a0fea0376#r59980214>
>